core/src/test/kotlin/io/specmatic/core/config/SpecmaticConfigAllTest.kt

```diff
@@ -146,4 +151,57 @@ internal class SpecmaticConfigAllTest {
 
         assertThat(parsedJSON(contractsJson)).isEqualTo(parsedJSON(expectedContractsJson))
     }
+
```

Comment:
This needs tests checking that the config can be deserialized when `consumes` or `provides` are absent.